src/Payments/CardPayment.res

```diff
@@ -152,8 +153,13 @@ let make = (
       defaultCardBody
     }
     if confirm.doSubmit {
-      let validFormat = (isBancontact || complete) && areRequiredFieldsValid
-      if validFormat && (showFields || isBancontact) && isCardBrandValid {
+      let validFormat =
+        (isBancontact ||
+        (isCVCValid->Option.getOr(false) &&
+        isCardValid->Option.getOr(false) &&
+        isCardSupported->Option.getOr(false) &&
```

Comment:
I think you can add this inside complete itself. Complete variable already contains whether card, cvc and expiry are valid

Reply:
Have added it separately